setonix/environments/env_apps/spack.yaml

```diff
@@ -19,23 +19,14 @@ spack:
       +user-mofff +user-netcdf +user-omp +user-phonon +user-plumed +user-ptm +user-qtb
       +user-reaction +user-reaxc +user-sdpd +user-smd +user-smtbq +user-sph +user-tally
       +user-uef +user-yaff ^kokkos@3.4.01+hwloc+memkind+numactl+openmp+tuning ^plumed@2.7.2
-
-    
-# TODO: charmpp+namd yet to be made to work with Shasta/Slingshot
-    - namd@2.14 interface=python fftw=3 ^charmpp@7.0.0
-      backend=mpi
-
-    
+      ^python@3.9.7+optimizations
+    - namd@2.14 interface=python fftw=3 +plumed ^python@3.9.7+optimizations ^charmpp@6.10.2 backend=ofi
```

Comment:
@halfflat you can get rid of the occurrences of `^python@3.9.7+optimizations`, it is not enforced in `packages.yaml` once and for all packages :)

Reply:
I agree with @marcodelapierre.